engine/src/main/java/org/terasology/logic/ai/HierarchicalAISystem.java

```diff
@@ -59,6 +59,9 @@
 
     // TODO add way to recognize if attacked
 
+    /**
+     * @param delta The time (in seconds) since the last engine update.
```

Comment:
Here you do not need to add JavaDoc as the method is overriding the `update` from `UpdateSubscriberSystem` which has a JavaDoc statement on it. That statement will automatically apply for child methods like this one.